tests/Cache/FileHandlerTest.php

```diff
@@ -26,6 +27,8 @@
  */
 final class FileHandlerTest extends TestCase
 {
+    use FileSystemAssertTrait;
```

Comment:
not on 5.3,
which functionality of this trait is used in this file ?

Reply:
`$this->assertFilePermissionMask(0666, $file);`

https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3123/files/cda7bf908fcbb0aa41e26dfe2dd5ecdedc8e61bc#diff-953496b5de42e8f87de0524f42d9cbcaR216

I can refactor to use the constraint directly to support 5.3

Reply:
Please do, we shall fix it on 2.2 without raising php engine